docs/usage/examples/opentelemetry.md

```diff
@@ -0,0 +1,233 @@
+
+#### Calls metric
+
+At first there are the `calls_total` metrics which display how often specific trace spans have been observed.
+
+For example:
+`calls_total{operation="renovate repository",service_name="renovate",span_kind="SPAN_KIND_INTERNAL",status_code="STATUS_CODE_UNSET"} 3` signals that 3 repositories have been renovated.
+`calls_total{operation="run",service_name="renovate",span_kind="SPAN_KIND_INTERNAL",status_code="STATUS_CODE_UNSET"} 1` represents how often Renovate has been run.
+
+If we combine this using the PrometheusQueryLanguage ( PromQL ), we can calculate the average count of repositories each Renovate run handles.
+
+```
+calls_total{operation="renovate repository",service_name="renovate"} / calls_total{operation="run",service_name="renovate"}
+```
+
+This metrics is also for spans generated by http calls:
+
+```yaml
+calls_total{http_host="registry.terraform.io:443",http_method="GET",http_status_code="200",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_UNSET"} 5
+```
+
+#### Latency buckets
+
+The second class of metrics exposed are the latency focused latency buckets which allow to create [heatmaps](https://grafana.com/docs/grafana/latest/basics/intro-histograms/#heatmaps).
+A request is added to a backed if the latency is bigger than the bucket value (`le`). `request_duration => le`
+
+As an example if we receive a request which need `1.533s` to complete get following metrics:
+
+```
+latency_bucket{http_host="api.github.com:443",le="0.1"} 0
+latency_bucket{http_host="api.github.com:443",le="1"} 0
+latency_bucket{http_host="api.github.com:443",le="2"} 1
+latency_bucket{http_host="api.github.com:443",le="6"} 1
+latency_bucket{http_host="api.github.com:443",le="10"} 1
+latency_bucket{http_host="api.github.com:443",le="100"} 1
+latency_bucket{http_host="api.github.com:443",le="250"} 1
+latency_bucket{http_host="api.github.com:443",le="9.223372036854775e+12"} 1
+latency_bucket{http_host="api.github.com:443",le="+Inf"} 1
+latency_sum{http_host="api.github.com:443"} 1.533
+latency_count{http_host="api.github.com:443"} 1
+```
+
+Now we have another request which this time takes 10s to complete:
+
+```
+latency_bucket{http_host="api.github.com:443",le="0.1"} 0
+latency_bucket{http_host="api.github.com:443",le="1"} 0
+latency_bucket{http_host="api.github.com:443",le="2"} 1
+latency_bucket{http_host="api.github.com:443",le="6"} 1
+latency_bucket{http_host="api.github.com:443",le="10"} 2
+latency_bucket{http_host="api.github.com:443",le="100"} 2
+latency_bucket{http_host="api.github.com:443",le="250"} 2
+latency_bucket{http_host="api.github.com:443",le="9.223372036854775e+12"} 2
+latency_bucket{http_host="api.github.com:443",le="+Inf"} 2
+latency_sum{http_host="api.github.com:443"} 11.533
+latency_count{http_host="api.github.com:443"} 2
+```
```

Comment:
I think that the conversation in https://github.com/renovatebot/renovate/pull/15731/files#r894133979 is still relevant and unresolved, and I believe it also applies for these lines here.

Reply:
IMO it makes to put here full example as these are the default buckets which user get, if using the `metricsprocessor`.

Reply:
Do users of this feature need to know what data gets output in order to use the feature?

Reply:
Not to use it, but to decide on its usefulness.

Reply:
Wouldn't a single image speak more than 22 lines of output?

Reply:
If we are talking about the same content than images are taking more space and harder to scale for browsers.